domains/brantes.json

```
@@ -0,0 +1,10 @@
+
+{
+  "owner": {
+    "username": "pedrobrantes",
+    "email": "58346706+PedroBrantes@users.noreply.github.com"
```

you can remove the email if you dont want to share it
```suggestion
{
  "owner": {
    "username": "pedrobrantes"
```